doc/flame/components.md

```diff
@@ -364,6 +364,20 @@ this.player = SpriteAnimationComponent.fromFrameData(
 If you are not using `FlameGame`, don't forget this component needs to be updated, because the
 animation object needs to be ticked to move the frames.
 
+To listen when the animation is done (when it reaches its last frame and is not looping) you can
+use `animation.completed`.
+
+For example, you can:
```

Comment:
```suggestion
Example:
```
A sentence can't end with "you can" :)

Reply:
Good catch, I was thinking "For example, you can do as follows:" but forgot to actually write it down haha.